src/Chapter/DataStructures.elm

```diff
@@ -4,10 +4,120 @@ data =
   { title = "Data Structures"
   , content = """
 
+Up until this point, you've used lots of different values of lots of different types, but have only had one way of storing multiple values, namely: lists. A list is an example of an extremely simple data structure. In this section, we'll take a look at some of the other data structures available to you in elm.
+
 # Union Types
 
+<dfn>Union types</dfn> are essentially a way to join different types together, hence their name. They allow you to store many values of different types inside of a single data structure. For those of you with a java background; as the <a href="http://elm-lang.org/learn/Union-Types.elm">Elm website</a> puts it: "Think of [union types] as enums on steroids."
+
+Let's start off with defining a very simple union type:
+
+```elm
+type Animal
+  = Rabbit
+  | Dog
+  | Cat
```

Comment:
I'm tempted to put simple ones like this on one line because it looks better, but I could be convinced otherwise (especially since we add to them later).